drake/CMakeLists.txt

```diff
@@ -143,6 +143,11 @@ endif()
 
 enable_testing()
 include(CTest)
+find_package(GTest REQUIRED)
+include_directories(${GTEST_INCLUDE_DIRS})
+if (GTEST_FOUND)
+  set (CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -pthread")
```

Comment:
perhaps adding it to GTEST_BOTH_LIBRARIES?  it also might want to be more platform specific (windows doesn't have pthread...?)